### DISCUSSION TITLE
Is it possible to turn the display off using GCODE?

### DISCUSSION THREAD

**Discussion by Starker3** [Jyers/Marlin#1444] [Q&A] [answered]
I've got my printer setup with Octoprint and a smart wifi plug to control the power on/off
I'd love to be able to turn the display off using a command from Octoprint. I haven't checked in the code exactly how this firmware handles controlling the display on/off.

I've tried using the M256 command documented here: https://marlinfw.org/docs/gcode/M256.html
But haven't had any luck with it

**Comment by Starker3** [comment, marked as answer]
Looks like this uses M251 in Jyers: https://github.com/Jyers/Marlin/blob/Ender_3_V2_JyersUI/Marlin/src/gcode/lcd/M251.cpp

**Comment by corben2** [comment]
This is the first I've heard that the official Marlin added this. They committed it August 1st, well after it was added to this fork. Maybe this isn't the place to discuss this, but we may want to switch from M251 to M256 to match them, and then add an "alias" on M251 that simply calls M256 for backwards compatibility?
